x/swap/simulation/params.go

```diff
@@ -0,0 +1,13 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+// ParamChanges defines the parameters that can be modified by param change proposals
+// on the simulation
+func ParamChanges(r *rand.Rand) []simulation.ParamChange {
+	return []simulation.ParamChange{}
```

Comment:
We should define this function to allow pools and swap fee to be changed. https://github.com/cosmos/cosmos-sdk/blob/v0.39.2/x/mint/simulation/params.go#L22 has good example

Reply:
https://github.com/Kava-Labs/kava/pull/924/commits/71d0eab53138120edc4ebecbd521af0721266184